cafebabel/tests/test_articles.py

```diff
@@ -5,7 +5,7 @@
 from flask import url_for
 from flask.helpers import get_flashed_messages
 
-from ..articles.models import Article
+from ..articles.models import Article, Tag
```

Comment:
Add `Tag.drop_collection()` in `pytest_runtest_teardown`

Reply:
Done.